tox.ini

```diff
@@ -32,8 +32,8 @@ envdir = .tox/docs
 passenv = DOCS_FROM_MASTER, QISKIT_DOCS_BUILD_TUTORIALS
 deps =
   -r requirements-dev.txt
-allowlist_externals touch
 commands =
+  allowlist_externals touch
   touch {toxinidir}/docs/.buildingdocs
   sphinx-build -b html {posargs} {toxinidir}/docs/ {toxinidir}/docs/_build/html
```

Comment:
```suggestion
allowlist_externals = touch
commands =
  touch {toxinidir}/docs/.buildingdocs
  sphinx-build -b html {posargs} {toxinidir}/docs/ {toxinidir}/docs/_build/html
```

Reply:
https://github.com/Qiskit/qiskit-metal/pull/893/commits/bfe28e7fd4073283e184b80d9be52e1bf85611bc

I did try the syntax that you proposed.  That had also failed.

Reply:
I do appreciate your help.  Thank you!!

Reply:
You forgot the `=` / the equals sign.

I suggested the correct fix - as a code snippet. On GitHub you could even click on a button and accept it - you do not need to update the code yourself. If you want to update the code yourself, please make sure to copy/paste it correctly. And :crossed_fingers: that it works soon. It is one of these days :-)